kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java

```diff
@@ -423,6 +499,27 @@ public void testGetProcessInstance() throws Exception {
         }
 
     }
+    
+        
+    public void testGetProcessInstanceWrongContainer() {
+        
+        Map<String, Object> parameters = new HashMap<String, Object>();
```

Comment:
Explicit type parameters are not required
```suggestion
        Map<String, Object> parameters = new HashMap<>();
```